docker/kuberay-autoscaler/Dockerfile

```diff
@@ -4,5 +4,6 @@ RUN $HOME/anaconda3/bin/pip uninstall -y ray
 COPY $WHEEL_PATH .
 RUN $HOME/anaconda3/bin/pip --no-cache-dir install "$WHEEL_PATH"[all]
 
+COPY run_autoscaler_with_retries.py /home/ray/run_autoscaler_with_retries.py
```

Comment:
This line looks like a duplicate with the line two below

Reply:
Indeed, I had meant to copy "autoscaling_config.py".